app/src/test/java/org/simple/clinic/patientcontact/PatientContactUiRendererTest.kt

```diff
@@ -107,10 +114,41 @@ class PatientContactUiRendererTest {
     uiRenderer.render(defaultModel().overdueAppointmentLoaded(None))
 
     // then
+    verifyStandardInteractions()
+
     verify(ui).hideCallResultSection()
     verifyNoMoreInteractions(ui)
   }
 
+  @Test
+  fun `if the secure call feature is enabled, show the secure call button and message`() {
+    // when
+    val model = defaultModel(phoneMaskFeatureEnabled = true)
+    uiRenderer.render(model)
+
+    // then
+    verify(ui).showSecureCallButton()
+    verify(ui).showSecureCallMessage()
+    verifyNoMoreInteractions(ui)
+  }
+
+  @Test
+  fun `if the secure call feature is disabled, hide the secure call button and message`() {
+    // when
+    val model = defaultModel(phoneMaskFeatureEnabled = false)
+    uiRenderer.render(model)
+
+    // then
+    verify(ui).hideSecureCallButton()
+    verify(ui).hideSecureCallMessage()
+    verifyNoMoreInteractions(ui)
+  }
+
+  private fun verifyStandardInteractions() {
```

Comment:
Rename to `hideSecureCallUi()`?

Reply:
I am not sure if we should even extract these to a separate method. 
Repetition in tests is never a problem and it is better to avoid a common pitfall while writing tests which is dependency of tests on each other.

Reply:
My intention behind doing this was to separate the assertions that are not specific to the scenario being testing in the specific test outside so that it is clear when reading the test what assertions are part of this test vs what are not.

This is also why I called it `verifyStandardInteractions` since I do not want to expose what exactly the standard interactions are.

Does this make sense?

Reply:
The way I see it, this has two issues:
- There is nothing like a standard interaction here and we shouldn't assume this as a standard interaction for the screen. This is not like an error or unexpected state.
- By separating out this verification, we are assuming that this is not something we care about in this particular test which isn't true. Testing whether this view is hidden or not is equally important part of Ui rendering for this state. If it isn't then we shouldn't be testing this at all.

Reply:
Hmm, I see this point as well. I'll inline this method into the rest of the tests for now.